examples/ising/ising/model.py

```diff
@@ -0,0 +1,47 @@
+import mesa
+import numpy as np
+
+from .spin import Spin
+
+
+class IsingModel(mesa.Model):
+    def __init__(
+        self,
+        width=50,
+        height=50,
+        spin_up_probability: float = 0.7,
+        temperature: float = 1,
+    ):
+        super().__init__()
+        self.temperature = temperature
+        self.grid = mesa.space.SingleGrid(width, height, torus=True)
+
+        for contents, (x, y) in self.grid.coord_iter():
+            cell = Spin((x, y), self, Spin.DOWN)
+            if self.random.random() < spin_up_probability:
+                cell.state = cell.UP
+            self.grid.place_agent(cell, (x, y))
+
+        self.running = True
+
+    def step(self):
+        agents_list = list(self.agents)
+        self._steps += 1000
```

Comment:
If I understand this correctly, you are using _step to count the number of agent activations (you use 1000, as in the for loop). Why use `_step` for this?